code/game/machinery/syndicatebeacon.dm

```diff
@@ -102,119 +102,163 @@
 		selfdestructing = 1
 		spawn() explosion(src.loc, 1, rand(1,3), rand(3,8), 10)
 
-
-
-#define SCREWED 32
-
-/obj/machinery/singularity_beacon //not the best place for it but it's a hack job anyway -- Urist
-	name = "ominous beacon"
-	desc = "This looks suspicious..."
+//Not the best place for it but it's a hack job anyway -- Urist
+/obj/machinery/singularity_beacon
+	name = "singularity beacon"
+	desc = "A suspicious-looking beacon. It looks like one of those snazzy state-of-the-art bluespace devices."
 	icon = 'icons/obj/singularity.dmi'
 	icon_state = "beacon"
-
 	anchored = 0
 	density = 1
-	layer = MOB_LAYER - 0.1 //so people can't hide it and it's REALLY OBVIOUS
-	stat = 0
+	machine_flags = WRENCHMOVE | FIXED2WORK
+
+	light_color = LIGHT_COLOR_RED
+	light_range_on = 2

```

Comment:
*bangs head into desk repeatedly.

These variables are the variables used if the machine has `use_auto_lights` set to nonzero, and it's not `BROKEN` or `NOPOWER`.